smoker/client/cli.py

```diff
@@ -334,8 +415,17 @@ def main():
     # Add status filter
     filters.append(('status', states))
 
+    discovered_hosts = _host_discovery(args)
+    hosts = ['localhost']
```

Comment:
It should not be default in this way. Sometimes you have installed only client with no running smokerd on localhost.